### ISSUE TITLE
Bugged Quest Item - "Washed Ashore" Quest

### ISSUE THREAD

**Issue by Allelya** [dalaranwow/dalaran-wow#4010]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: When attempting to retrieve the quest item of "bones" from the creature on the shore for the quest "Washed Ashore" in zone Darkshore, all I receive is a popup window saying Greetings. Cannot loot the quest item.

**Expected behaviour**: When I click the creature (when the mouse turns into a gear) I should receive the quest loot.

**Steps to reproduce the problem**:

1. Accept quest "Washed Ashore" from Gwyneth Bly'leggonde
2. Go to area on shore where half bones dead thresher lies.
3. Click on creature to loot.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills** 
All NPCs included were mentioned.
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**
